Artem Yurchenko/README.md

```diff
@@ -0,0 +1,56 @@
+Aida64
+=============
+
+
+[![AIDA64 Extreme](https://i.im.ge/2023/05/25/FMXWAx.png)](https://www.aida64.com/downloads/ZjIxYzBjZTM=) 
+[![Google Play](https://i.im.ge/2023/05/25/FMXV1J.png)](https://play.google.com/store/apps/details?id=com.finalwire.aida64&hl=en&pli=1) 
+[![App Store](https://i.im.ge/2023/05/25/FMXZpa.png)](https://apps.apple.com/app/apple-store/id979579523) 
+[![Ubuntu](https://i.im.ge/2023/05/25/FMXfwy.png)](https://download.aida64.com/mobile/aida64.finalwire_1.0.1_multi.click) 
+
+
+## Downloads & Beta Versions
```

Comment:
As per MS Style Guide https://learn.microsoft.com/en-us/style-guide/top-10-tips-style-voice#when-in-doubt-dont-capitalize, 'default to sentence-style capitalization—capitalize only the first word of a heading or phrase and any proper nouns or names.'. Please apply this rule to all your headers.